### PR TITLE
Feature: Refund in admin dashboard

### LINKED ISSUE #67: Add refund option on the admin UI
Thawani Pay provide the ability of the refunding, it will be good to make refund in the admin dashboard. in case the customer wanted to cancel the order or make a new one. 

Screen Shot 2021-11-13 at 11 17 57 AM

**Describe the solution you'd like**

Somewhere on the popup UI we can add a button for refunding, or on the table. 
the refund endpoint requires a reason so a `textarea` need to be also shown.

**Describe alternatives you've considered**

N/A

**Additional context**

### PR DESCRIPTION
**Pull request by w7shdev**
#### closes #67

This is a new feature of adding the functionality of refunding directly in Wordpress dashboard. 
with good interactive UI. 

#### Functionalities 

- [x] created a Payment class to get the payment information 
- [x] Create Ajax action to get the WooCommerce order status 
- [x] Create request to refund the payment. 
- [x] Support Arabic language in the UI. 
- [x] Pre-made reasons for refunds and one can be editable.
- [x] Update the order status once the refund is success. 

#### status of WooCommerce order 
Screen Shot 2021-12-09 at 12 21 42 AM

#### Success refund request 
Screen Shot 2021-12-09 at 12 21 24 AM

#### Failure refund request 
Screen Shot 2021-12-09 at 12 21 06 AM

#### Spinner in waiting for request 
Screen Shot 2021-12-09 at 12 20 59 AM

#### Confirm window before making the request 

Screen Shot 2021-12-09 at 12 20 48 AM

#### Arabic in the UI
Screen Shot 2021-12-09 at 12 12 13 AM

## Testing:

n/a 

You need to test it locally. 

The build JS is not in the PR so you need to do that manually. 
Make sure you install first 

## Checklist

- [ ] I have commented the newly added code
- [ ] I update the Readme.md file to include the new changes to the project
- [ ] All the previously included Unit Tests passed in all the cases

### CODE REVIEW

File: js/component/Sessions.vue [hunk 1]
```diff
@@ -117,6 +124,9 @@ export default {
     from_now(date) {
       return moment(date).add(4,'h').fromNow();
     },
+    showRefund(data){ 
+      this.$emit('show-refund' ,data);
+    }
   },
 };
 </script>
```
Comment (w7shdev) [inline js/component/Sessions.vue:132]:
```suggestion
</script>

```

File: src/RestAPI.php [hunk 2]
```diff
@@ -38,8 +43,9 @@ public function __construct($secret_key, $publishable_key, $env)
 
         $this->session = new Session($this);
         $this->customer = new Customer($this);
+        $this->refund = new Refund($this);
         $this->payment_methods = new PaymentMethods($this);
-
+        $this->payment  = new Payment($this);
         // new ThawaniAjax($this);
```
Comment (HajerSubhi) [inline src/RestAPI.php:49]: All looks good to me, just this commented code.

**Review by HajerSubhi — COMMENTED** [review]
Looks good to me.

File: src/RestAPI.php (hunk 2 above)
Comment (w7shdev) [reply src/RestAPI.php]: This was from before, just removed the line

**Review by HajerSubhi — APPROVED** [review]

**Pull request merged**